### ISSUE TITLE
Fix find request by id

### ISSUE THREAD

**Issue by jbonnet** [sonata-nfv/son-gkeeper#819]
```
D, [2017-08-18T13:56:38.146919 #8] DEBUG -- :   [1m[35mRequest Load (10.7ms)[0m  SELECT  "requests".* FROM "requests" WHERE "requests"."id" = $1 LIMIT 1  [["id", "69ec1135-e556-4223-bd3b-ce8d7a289656"]]
2017-08-18 13:56:38 - ActiveRecord::RecordNotFound - Couldn't find Request with 'id'=69ec1135-e556-4223-bd3b-ce8d7a289656:
...
```

**Comment by jbonnet** [comment]
Duplicated from #801

**Comment by jbonnet** [comment]
Closed by #887
